src/values/length.rs

```diff
@@ -99,6 +193,10 @@ macro_rules! define_length_units {
         $(#[$meta])*
         $name(CSSNumber),
       )+
+
+      /// A [`<length>`](https://www.w3.org/TR/css-values-4/#lengths) value
+      /// without a unit specified.
+      Unitless(CSSNumber),
```

Comment:
What is this needed for? Aren't unitless lengths only allowed for zero, and compute to pixels? So why not just store as pixels?